pom.xml

```diff
@@ -136,7 +137,7 @@
     <maven-dependency-plugin.version>2.8</maven-dependency-plugin.version>
     <maven-deploy-plugin.version>2.7</maven-deploy-plugin.version>
     <maven-enforcer-plugin.version>1.2</maven-enforcer-plugin.version>
-    <maven-failsafe-plugin.version>2.13</maven-failsafe-plugin.version>
+    <maven-failsafe-plugin.version>3.0.0-M3</maven-failsafe-plugin.version>
```

Comment:
What does `M3` mean here?

Reply:
It is just a part of their version number. Probably a patch number?  I am using this version because it is documented in their examples.